service/src/main/java/greencity/security/service/LoginAttemptServiceImpl.java

```diff
@@ -0,0 +1,65 @@
+package greencity.security.service;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoginAttemptServiceImpl implements LoginAttemptService {
+    private final LoadingCache<String, Integer> attemptsCache;
+    private final HttpServletRequest request;
+    @Value("${bruteForceSettings.maxAttempts}")
+    private int maxAttempt;
+
+    public LoginAttemptServiceImpl(HttpServletRequest request,
+        @Value("${bruteForceSettings.blockTimeInHours}") long blockTimeInHours) {
+        super();
```

Comment:
what is the reason for calling `super()` here?

Reply:
I agree, it makes no sense
I fixed that